src/DotNetWorker.Core/Context/Features/DefaultFunctionInputBindingFeature.cs

```diff
@@ -55,8 +55,18 @@ public async ValueTask<FunctionInputBindingResult> BindFunctionInputAsync(Functi
 
                     TryGetBindingSource(parameter.Name, functionBindings, out object? source);
 
+                    // Option 1: Check if source is an empty string and parameter is reference or nullable type
+                    if (source is string str && string.IsNullOrEmpty(str))
```

Comment:
I think this is my preferred approach. Regardless of the binding type, source is always a string that needs to be convertered into a Type. So I think this is a safe change